src/main/kotlin/no/nav/helsearbeidsgiver/bro/sykepenger/domene/ForespurtDataDto.kt

```diff
@@ -19,7 +19,7 @@ sealed class ForespurtDataDto
 
 @Serializable
 @SerialName("Arbeidsgiverperiode")
-data class ArbeidsgiverPeriode(val forslag: List<Periode>) : ForespurtDataDto()
+object ArbeidsgiverPeriode : ForespurtDataDto()
 
```

Comment:
Vil Dette være bakoverkombatibelt i databasen

Reply:
?

Reply:
Ja, siden dben lagrer i jsonb. Da trenger vi ikke endre noe for å lagre, og vi leser json-en rett inn i et ForespurtDataDto-objekt, som overser overflødige felter i json-en.